jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInitProjectWizardTest.kt

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.execution.sam
+
+import com.intellij.ide.projectWizard.ChooseTemplateStep
+import com.intellij.ide.projectWizard.NewProjectWizardTestCase
+import com.intellij.ide.projectWizard.ProjectSettingsStep
+import com.intellij.ide.projectWizard.ProjectTypeStep
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.projectRoots.JavaSdk
+import com.intellij.openapi.projectRoots.ProjectJdkTable
+import com.intellij.openapi.projectRoots.SdkType
+import com.intellij.testFramework.IdeaTestUtil
+import com.jetbrains.python.sdk.PythonSdkType
+import software.amazon.awssdk.services.lambda.model.Runtime
+import software.aws.toolkits.jetbrains.settings.SamSettings
+import software.aws.toolkits.jetbrains.ui.wizard.SamInitModuleBuilder
+import software.aws.toolkits.jetbrains.ui.wizard.SamInitTemplateSelectionStep
+import software.aws.toolkits.jetbrains.utils.rules.PyTestSdk3x
+import kotlin.reflect.KClass
+
+// JUnit3-style to take advantage of IDEA's test base
+class SamInitProjectWizardTest : NewProjectWizardTestCase() {
+    fun testExceptionIfSamNotConfigured() {
+        try {
```

Comment:
We should still be able to use the AssertJ `assertThrownBy` for this rather than a try/catch : http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion